### PR TITLE
Add `middleman_protocol` app to concent container

### PR DESCRIPTION
**Pull request by bartoszbetka**

### CODE REVIEW

File: containers/concent-api/install-concent-api.sh.j2 [hunk 1]
```diff
@@ -8,3 +8,7 @@ source "$virtualenv_dir/bin/activate"
 python -m pip install --upgrade pip
 python -m pip install --requirement "/srv/http/concent_api/requirements.lock"
 python -m pip install gunicorn=="{{ gunicorn_version }}"
+(
+    cd "/srv/http/middleman_protocol/"
+    python setup.py develop
+)
```
Comment (cameel) [inline containers/concent-api/install-concent-api.sh.j2]: Offtopic: we should be clearing pip cache after installation in this script. You could do it in a separate pull request.

Comment (cameel) [inline containers/concent-api/install-concent-api.sh.j2]: Please create a separate script. We don't want to have to reinstall other dependencies when the source changes.

File: containers/package-builder/build-virtualenv.sh.j2 [hunk 2]
```diff
@@ -8,3 +8,7 @@ mkdir --parents "$build_dir/virtualenv/"
 virtualenv --python "python{{ python_version }}" "$build_dir/virtualenv/"
 source "$build_dir/virtualenv/bin/activate"
 python -m pip install -r "$requirements_path"
+(
+    cd "$build_dir/repositories/concent/middleman_protocol/"
+    python setup.py develop
+)
```
Comment (cameel) [inline containers/package-builder/build-virtualenv.sh.j2]: Same as above. Put it in a separate script so that you don't have to change the order in the `Dockerfile`.

File: containers/concent-api/install-concent-api.sh.j2 (hunk 1 above)
Comment (bartoszbetka) [reply containers/concent-api/install-concent-api.sh.j2]: Ok. I will make it in separate pull request.

Comment (bartoszbetka) [reply containers/concent-api/install-concent-api.sh.j2]: Done

File: containers/concent-api/install-concent-api.sh.j2 [hunk 3]
```diff
@@ -6,6 +6,7 @@ virtualenv_dir=/srv/http/virtualenv
 virtualenv --python "python{{ python_version }}" "$virtualenv_dir"
 source "$virtualenv_dir/bin/activate"
 python -m pip install --upgrade pip
+cd /srv/http/concent_api/
```
Comment (cameel) [inline containers/concent-api/install-concent-api.sh.j2]: Please add a comment that requirements needs to be installed from the directory it's in because it contains relative paths.

File: containers/Makefile [hunk 4]
```diff
@@ -110,20 +110,20 @@ build/concent-api/requirements.lock: build/concent-api/concent-api.tar
 build/concent-api/testing.pref: ../concent-builder/files/testing.pref
 	cp "$<" "$@"
 
-build/concent-api/concent-api.tar: build/repositories/concent/  build/virtualenv/
+build/concent-api/concent-api.tar: build/repositories/concent  build/virtualenv/
```
Comment (cameel) [inline containers/Makefile]: Cosmetic changes like this should be in a separate commit, especially when there's more of them like in this case.

File: containers/concent-api/install-concent-api.sh.j2 (hunk 3 above)
Comment (bartoszbetka) [reply containers/concent-api/install-concent-api.sh.j2]: Done.

File: containers/Makefile (hunk 4 above)
Comment (bartoszbetka) [reply containers/Makefile]: Done.
